cmd/common/utils/flags.go

```diff
@@ -30,6 +31,69 @@ const (
 
 var SupportedOutputModes = []string{OutputModeColumns, OutputModeJSON, OutputModeCustomColumns}
 
+type CommonFlags struct {
+	// Containername allows to filter containers by name.
+	Containername string
```

Comment:
I suggest not adding this here. It will diverge between `kubectl-gadget` and `local-gadget` after https://github.com/inspektor-gadget/inspektor-gadget/issues/644 is implemented.